grammars/hrm.cson

```diff
@@ -32,29 +32,25 @@
         'name': 'keyword.control.c'
   }
   {
-    'match': '^    (COMMENT) +([0-9]+)'
+    'match': '^ *(COMMENT) +([0-9]+)'
     'captures':
       '1':
         'name': 'entity.name.function.preprocessor.c'
       '2':
-        'name': 'string.quoted.double.include.c'
+        'name': 'constant.numeric.c'
```

Comment:
Cool.  So, the only remaining thing I'ld like (and I'ld be happy to do this when I land the patch if you don't feel like it) is to colour the number after `DEFINE COMMENT` the same as the number after `COMMENT`, so that it ties them together visually a little…  Would you like to do that, or shall I?